docs/application/dotnet/guides/multimedia/thumbnail-images.md

```diff
@@ -27,17 +27,43 @@ using Tizen.Multimedia.Util;
 > **Note**   
```

Comment:
@hsgwon request you change the note to maintain consistency with other pages, 
Current:
>**Note** 
> The input media file can be common content in the device storage (internal or external) or private content in your application data.

Add ">" in middle after **Note** (Global) applies to other **Note** too.
> **Note**  
>
>The input media file can be common content in the device storage (internal or external) or private content in your application data.

Reply:
Did you mean like below statement?

```
> **Note**

-->

> **Note** >
```

Reply:
@hsgwon no not like this:
We use the following styling for notes: 

>**Note**
>
>Your note text.......

So in current notes present in the page have the following styling: 
>**Note**
>Your note text starts here 

The difference is an extra **>** followed by line break  between 
>**Note** and >Your note text starts here. 

Reason: It essentially adds a line break such that the note text is easily legible.   


For more information on **Note** styling, please see https://github.com/Samsung/tizen-docs/blob/master/styleguide/template-guide.md#note

Reply:
Did you see it in tizen.org? 
There's no problem.
https://developer.tizen.org/development/guides/.net-application/media-and-camera/thumbnail-images

Reply:
@hsgwon thanks for your time and patience. 
I agree it looks fine but there is no spacing between **Note** and the **Note text**. It would look better visually if we insert a ">" with a linebreak. And for the sake of consistency throughout the Tizen website, we are using the mentioned style, which is the agreed-upon standard of Tizen Docs.

If you don't think you need to fix it, that's fine.

Cc: @mijin85cho

Reply:
As I see, there's no difference between them.
Please check below link. It's written as you said.

https://developer.tizen.org/development/guides/.net-application/application-management/applications/widget-application

Reply:
@mijin85cho , Can you please provide your viewpoint on this.

Reply:
If it's standard, of course, I'll follow it. 
Please let me know the standard document we should following.
I just want to know the standard to prevent further mistake.

Reply:
Hello.
For the line break after the \*\*Note**, we have 2 ways:

1. Adding one line with > 
   (as @nawab-samsung guided, with this link : https://github.com/Samsung/tizen-docs/blob/master/styleguide/template-guide.md#note)

   ```
   > **Note**
   >
   > Your note text.......
   ```

2. Adding 2 spaces after  \*\*Note**
   (As used in this page)

   ```
   > **Note**(space)(space)
   > Your note text.......
   ```

When we initiate the doc files to md format, we used both rules without choosing one rule.
That's the reason in this file that note view is not broken.
(The  \*\*Note** has 2 spaces after that.)

However, leaving 2 spaces after string sometimes caused error,
as some editing tools automatically removed the spaces after the sentence, considering it as a human error.

With that reason, we fixed the rule to use the empty line with >.
So @nawab-samsung's suggestion is correct.

@hsgwon, would you please change the note style as @nawab-samsung's comment?

Reply:
You did already mentioned about a style guide for this. 
OK I'll fix it.

Reply:
@mijin85cho , the style guide enforces just following. 
Please add additional information you said to clarity it.

```
should add "Note" with strong.
```

Reply:
@hsgwon In the style guide, we added the sample code of Note itself,
to guide how to set the Note form.

![image](https://user-images.githubusercontent.com/9958374/61854231-9cf5fd00-aef8-11e9-8ac1-675475b68bcb.png)

As it guides to use one empty line with >, I hope the reader can follow it.